open_academy/views/templates.xml

```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <data>
+<!--
```

Comment:
Same here. Don't add commented code. Please remove this file and all the references to it.